Stellar-ledger.x

```diff
@@ -506,12 +509,26 @@ struct LedgerCloseMetaExtV1
     int64 sorobanFeeWrite1KB;
 };
 
+struct LedgerCloseMetaExtV2
+{
+    int64 sorobanFeeWrite1KB;
+
+    uint32 currentArchivalEpoch;
+
+    // The last epoch currently stored by validators
+    // Any entry restored from an epoch older than this will
+    // require a proof.
+    uint32 lastArchivalEpochPersisted;
```

Comment:
How is this supposed to be used? I thought Core will tell when a proof is necessary.

Reply:
I don't think downstream will currently use it, but my goal here was for meta to have sufficient information such that somebody would be able to construct proofs without captive-core. It would take a lot of work and I don't think it's likely someone will actually do it, but with the curerntArchivalEpoch, the lastArchivalEpochPersisted, and the eviction meta, a downstream system would be able to maintain all merkle trees and construct restoration proofs correctly without using captive-core BucketListDB endpoints.